esp_custom_part_ota/examples/custom_part_ota_example/README.md

```diff
@@ -0,0 +1,29 @@
+# Custom Data Partition OTA
+
+Custom Data Partition OTA aims at enabling Over-the-Air data update for custom data partitions in ESP-IDF.
+
+## ESP Custom Partition OTA Abstraction Layer
```

Comment:
This example uses which scheme, should we mention about it here?

Reply:
I have added a config option to enable/disable the backing-up of the data